### PR TITLE
Add ipython_genutils.

### PR DESCRIPTION
**Pull request by pelson**

### CODE REVIEW

**Comment by msarahan** [comment]
LGTM - might suggest a slightly more descriptive summary, though.  What kind of utilities?

**Comment by ocefpaf** [comment]
> LGTM - might suggest a slightly more descriptive summary, though. What kind of utilities?

Hard to create a summary for a package that will disappear soon :smile: 

```
This package shouldn't exist. It contains some common utilities shared by Jupyter and IPython projects during The Big Split™. As soon as possible, those packages will remove their dependency on this, and this repo will go away.

No functionality should be added to this repository, and no packages outside IPython/Jupyter should depend on it.
```

**Comment by jakirkham** [comment]
In other words, "don't look at me, I'm genutils". ( http://blog.jupyter.org/2015/04/15/the-big-split/ )

File: recipes/ipython_genutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    - ipython_genutils.tests
+
+about:
+  home: http://ipython.org
+  license: BSD 3-Clause
+  summary: Vestigial utilities from IPython
+
+extra:
+  recipe-maintainers:
+    - minrk
```
Comment (minrk) [inline recipes/ipython_genutils/meta.yaml]: 👍

**Comment by minrk** [comment]
Vestigial is exactly the right term for genutils. It's the stuff we couldn't quite get rid of as part of the big split without spending too much time than we wanted to. Eventually, it will go away in favor of real packages that provide this functionality (six, etc.).

**Comment by jakirkham** [comment]
What's still WIP?

**Comment by jakirkham** [comment]
LGTM

@pelson, would it be ok to merge this? Is there something that still needs to be done here (commenting on the "WIP" in the title)?

**Comment by pelson** [comment]
Thanks for the nudges 😉

**Pull request merged**